src/index.ts

```diff
@@ -96,8 +99,16 @@ export class WaxJS {
     ) => void;
     metricURL?: string;
     returnTempAccounts?: boolean;
+    chainName?: string;
   }) {
     this.rpc = new JsonRpc(rpcEndpoint);
```

Comment:
this should be `this.registryRpc = new JsonRpc(rpcEndpoint);`

Reply:
updated base on comment 👍